tests/core-addon/integration/core_addon.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const firefox = require("selenium-webdriver/firefox");
+const { Builder, By, until } = require("selenium-webdriver");
+const assert = require("assert");
+
+let driver = null;

```

Comment:
I think this can be safely removed...
```suggestion
```

Reply:
Hm, this is there because `describe` sets up and tears it down so it's created in a different scope (`beforeEach` function). I can just make it `let driver;` and move into the `describe` function I guess?